core/services/run_executor.go

```diff
@@ -111,6 +112,7 @@ func (re *runExecutor) Execute(runID *models.ID) error {
 		if run.GetStatus().Errored() {
 			logger.Warnw("Task failed", run.ForLogger()...)
 		} else {
+			logger.Debug("val", run.Result.Data, run.Result.ErrorMessage)
```

Comment:
OK to leave this here but perhaps a bit more detail and use Debugw instead?

Reply:
oh woops didn't mean to leave this in at all.